distributed/tests/test_deadline.py

```diff
@@ -56,10 +57,19 @@ async def test_deadline_progress():
     )
 
 
+def test_deadline_expiration():
+    deadline = Deadline.after(0.1)
+    sleep(0.1)
+    assert deadline.expired is True
+    assert deadline.remaining == 0
```

Comment:
sometimes, windows is just messed up. it appears we cannot make this assertion on windows. I'll go ahead and relax this sleep as well